windows/buildDockerWin.ps1

```diff
@@ -0,0 +1,27 @@
+param ($registry, $version)
```

Comment:
can we add a comment/description that this is used by the GitHub Actions workflow?

Reply:
It's the same script used for building the images for local development, I'll add comments about this.